File: FrontEnd/src/app/models/monedasDeCuenta.ts
export class MonedasDeCuenta {
    
    idCuenta:number;
    idMoneda:number;
    unidades:number;
    saldoMoneda:number;

    constructor(idCuenta:number,idMoneda:number,unidades:number,saldoMoneda:number){
        this.idCuenta = idCuenta;
        this.idMoneda = idMoneda;
        this.unidades = unidades;
        this.saldoMoneda = saldoMoneda;
    }



}

File: BackEnd/CryptoPIL/Negocio/MonedasDeCuenta.cs
using System;
using System.Collections.Generic;

namespace CryptoPIL.Models
{
    public partial class MonedasDeCuenta
    {
        public int IdMonedasDeCuenta { get; set; }
        public int IdCuenta { get; set; }
        public int IdMoneda { get; set; }
        public double Unidades { get; set; }
        public decimal SaldoMoneda { get; set; }

        public virtual Cuenta IdCuentaNavigation { get; set; } = null!;
        public virtual Moneda IdMonedaNavigation { get; set; } = null!;
    }
}
